core/src/main/java/org/apache/carbondata/core/mutate/CarbonUpdateUtil.java

```diff
@@ -482,176 +482,6 @@ public boolean accept(CarbonFile file) {
 
   }
 
-  /**
-   * Handling of the clean up of old carbondata files, index files , delete delta,
-   * update status files.
-   * @param table clean up will be handled on this table.
-   * @param forceDelete if true then max query execution timeout will not be considered.
-   */
-  public static void cleanUpDeltaFiles(CarbonTable table, boolean forceDelete) throws IOException {
```

Comment:
I didn't see delta file cleanup from the trashManager, will there be any impact because of this ? we have to test with stale delta files once before removing I think.

Reply:
agree with @ajantha-bhat , please test and also this method used to take care of the aborted scenario, now how its being taken care?

Reply:
only keep cleanStaleDeltaFiles to handle exceptions.

clean files should work with the concurrent update, so remove it.

Reply:
cleanStaleDeltaFiles will be called only in case of exception here, please see below points
1. in Projectfordeletecommand, its called two times both in exception  case and finally block, thats wrong, it should be only in finally block
2. its called multiple times in `DeleteExecution ` class which needs to be checked and avoid it.
3. since its handled only in case of exception case, how its taken care in case of application crash or shutdown scenario?
4. cleanStaleDeltaFiles  had cases of handling the update and delete aborted case, how its handled now, as the complete method is removed?